### PR TITLE
Updates to euler bringup launch files

### PR DESCRIPTION
**Pull request by marip8**
- Remapped robot joint states from controller to new topic
- Added mutable joint states and robot joint states to source list of joint state publisher
- Performed manual calibration of ASUS camera

### CODE REVIEW

File: euler_support/urdf/euler_macro.xacro [hunk 1]
```diff
@@ -159,13 +164,13 @@
...
+      <child link="${prefix+robot_prefix}base_link"/>
       <origin xyz="0.067518 0 2.016" rpy="3.1415 0 0"/>
       <axis xyz="0 0 0"/>
     </joint>
     <!--motoman to end effector link-->
-    <joint name="${prefix}motoman_link_t_to_eef_base" type="fixed">
-      <parent link="${prefix}motoman_link_t"/>
+    <joint name="${prefix+robot_prefix}link_t_to_eef_base" type="fixed">
+      <parent link="${prefix+robot_prefix}link_t"/>
       <child link="${prefix}eef_eef_base"/>
```
Comment (jrgnicho) [inline euler_support/urdf/euler_macro.xacro]: I know you didn't add this but the end-effector name has a repeated "eef" term in it.  Could you fix that?

File: euler_support/urdf/euler_macro.xacro [hunk 2]
```diff
@@ -1,182 +1,175 @@
...
+    <joint name="${prefix}platform_to_arm" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}motoman_base_link"/>
+      <origin xyz="0.067518 0 2.016" rpy="3.1415 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <!--motoman to end effector link-->
+    <joint name="${prefix}motoman_link_t_to_eef_base" type="fixed">
+      <parent link="${prefix}motoman_link_t"/>
+      <child link="${prefix}eef_eef_base"/>
```
Comment (jrgnicho) [inline euler_support/urdf/euler_macro.xacro]: Out of curiosity, what did you use to format the xml?

**Comment by jrgnicho** [comment]
Looks good, please take a look at the eef link name comment and see if it's an easy fix.

File: euler_support/urdf/euler_macro.xacro (hunk 2 above)
Comment (marip8) [reply euler_support/urdf/euler_macro.xacro]: xmllint --format `old_file` > `new_file`

**Comment by jrgnicho** [comment]
@marip8 @rhwrd  thanks for the contributions.

**Pull request merged**